pyproject.toml

```diff
@@ -1,78 +1,85 @@
-[tool.poetry]
-name = 'skynet'
-version = '0.1a13'
-description = 'Decentralized compute platform'
-authors = ['Guillermo Rodriguez <guillermo@telos.net>']
-license = 'AGPL'
-readme = 'README.md'
+[project]
+name = "skynet"
+version = "0.1a13"
+description = "Decentralized compute platform"
+authors = [{ name = "Guillermo Rodriguez", email = "guillermo@telos.net" }]
+requires-python = ">=3.10,<3.13"
+readme = "README.md"
+license = "AGPL-3.0-or-later"
+dependencies = [
+    "pytz~=2023.3.post1",
+    "trio>=0.22.2,<0.23",
+    "Pillow>=10.0.1,<11",
+    "docker>=6.1.3,<7",
+    "py-leap",
+    "toml>=0.10.2,<0.11",
+    "msgspec>=0.19.0,<0.20",
+    "numpy<2.1",
+    "protobuf>=5.29.3,<6",
+    "zstandard>=0.23.0,<0.24",
+    "click>=8.1.8,<9",
+    "httpx>=0.28.1,<0.29",
+]
 
-[tool.poetry.dependencies]
-python = '>=3.10,<3.13'
-pytz = '^2023.3.post1'
-trio = '^0.22.2'
-Pillow = '^10.0.1'
-docker = '^6.1.3'
-py-leap = {git = 'https://github.com/guilledk/py-leap.git', rev = 'v0.1a32'}
-toml = '^0.10.2'
-msgspec = "^0.19.0"
-numpy = "<2.1"
-protobuf = "^5.29.3"
-zstandard = "^0.23.0"
-click = "^8.1.8"
-httpx = "^0.28.1"
+[project.scripts]
+skynet = "skynet.cli:skynet"
+txt2img = "skynet.cli:txt2img"
+img2img = "skynet.cli:img2img"
+upscale = "skynet.cli:upscale"
+inpaint = "skynet.cli:inpaint"
 
-[tool.poetry.group.frontend]
-optional = true
+[dependency-groups]
```

Comment:
ahh wait nm 😂 (per above)

yeah this looks good; i think i prefer this table of arrays vs. the table of vars - means you don't have to do the weird `'='.join()` on elements if parsing manually or wtv.